packages/app/package.json

```diff
@@ -6,6 +6,7 @@
     "lint": "eslint . --max-warnings 0",
     "lint:fix": "eslint . --fix",
     "build:app": "shx rm -rf dist && tsc -p . && webpack --config webpack.config.js && webpack --config webpack.preload.config.js && shx cp -r public dist/public",
+    "genPreload": "cross-env ts-node script/generatePreload.ts && npx prettier src/preloadApi.ts --write && npx prettier src/windowApi.d.ts --write && npm run lint:fix",
```

Comment:
do we need npx here? I think we have prettier in this project. Also lint:fix should be sufficient, as prettier is included in our eslint config?